### ISSUE TITLE
Translation missing *.yes & *.no

### ISSUE THREAD

**Issue by antododo** [loomnetwork/dc-translations#14]
Translation is missing for the Subscribe to Digest switch, **\*.yes** & **\*.no**:
![translation missing](https://user-images.githubusercontent.com/14948460/39061119-539428da-4491-11e8-9781-40716118941b.png)
This is an example in Deutsch but it's the same for all the languages.

`
<span class="switch-active" aria-hidden="true"><span class="translation_missing" title="translation missing: de.yes">Yes</span>`
`
<span class="switch-inactive" aria-hidden="true"><span class="translation_missing" title="translation missing: de.no">No</span></span>
`

**Comment by sekl** [comment]
Thanks @pl4ym0re , it's fixed in the translation files now, will be updated on the site soon. :)
